web/onos-gui/src/app/onos-config/change-value.util.spec.ts

```diff
@@ -84,15 +84,15 @@ describe('ChangeValueService', () => {
 
     it('Int value', () => {
         const ab = new ArrayBuffer(8);
-        const intArr = new Uint8Array(ChangeValueUtil.longToByteArray(1234));
+        const intArr = new Uint8Array(ChangeValueUtil.longToByteArray(12345678));
```

Comment:
I extracted the byte array from the go side and used it here as an input
```const intArr = new Uint8Array([0x4E,0x61,0xBC,0x0,0x0,0x0,0x0,0x0]);```
and it gave the same output as **longToByteArray()** (I wasn't sure it was correct)